phpunit.xml.dist

```diff
@@ -29,8 +28,23 @@
     </filter>
 
     <php>
+        <env name="GESTPAY_SHOP_LOGIN" value="GESPAYXXXX"/>
+        <env name="GESTPAY_API_KEY" value="api-key"/>
         <env name="STRIPE_API_KEY" value="api-key"/>
         <env name="ENABLE_API_INTEGRATION" value="false"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
     </php>
 
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
```

Comment:
do you really need it?

Reply:
This is mandatory for the ClockMock to work, i haven't tested it but it's said in the sf docs. However, i'm gonna check it and i'll tell you the results.

Reply:
Yeah, this is needed. We need to configure de namespaces for the ClockMock, without that it does not work properly.